### PR TITLE
change how hf is configured

### PR DESCRIPTION
**Pull request by akeym**
I feel like I'm missing something..

### CODE REVIEW

File: roles/overrides/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# special configs that are annoying to template and Splunk is just going to ruin them anyway.
+---
+
+- name: Copy custom props.conf
+  copy: src="{{splunk_repository.repository_root}}/conf/heavyforwarder/props.conf"
```
Comment (dubiousOne) [inline roles/overrides/tasks/main.yml]: path is wrong, there is no conf/heavyforwarder directory on ansible-vd01

File: roles/overrides/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- name: Copy custom transforms.conf
+  copy: src="{{splunk_repository.repository_root}}/conf/heavyforwarder/transforms.conf"
+        dest="/opt/splunk/etc/system/local/props.conf"
+        mode=0600
+  when: splunk_overrides.transforms_conf == True
+
+- name: Copy custom inputs.conf
+  copy: src="{{splunk_repository.repository_root}}/conf/heavyforwarder/inputs.conf"
+        dest="/opt/splunk/etc/system/local/props.conf"
```
Comment (dubiousOne) [inline roles/overrides/tasks/main.yml]: destination file is the same in each task, should match source filename

**Review by dubiousOne — CHANGES_REQUESTED** [review]

**Comment by akeym** [comment]
this branch has been re-worked, please review

**Comment by dubiousOne** [comment]
good 2 go

**Review by dubiousOne — APPROVED** [review]

**Pull request merged**